pkg/bootstrapper/bootstrapper.go

```diff
@@ -87,7 +87,7 @@ func WithRetrier(retrier *retrier.Retrier) BootstrapperOpt {
 	}
 }
 
-func (b *Bootstrapper) DeleteBootstrapCluster(ctx context.Context, cluster *types.Cluster, isUpgrade bool) error {
+func (b *Bootstrapper) DeleteBootstrapCluster(ctx context.Context, cluster *types.Cluster, operationType constants.Operation, isForceCleanup bool) error {
```

Comment:
For readability mainly. Wherever DeleteBootstrapCluster is called, it is better to see the operationType being passed rather than a 'true' or 'false'. Also, wanted to bring up the discussion around what force cleanup for Delete operation would mean as far as deleting the bootstrap cluster is concerned.